### ISSUE TITLE
Having trouble building...

### ISSUE THREAD

**Issue by 0xc000005a** [xoofx/CppAst#10]
I'm trying to build the project on Windows 7 with VS2017 or 2019 with a few issues.
First, on the CppAst project no a target framework is set, and the option is grayed out and can't be set anyway. So I created a new project using the original sources and .NET 4.7.2. There I added ClangSharp 3.8.0, libclang 9.0 etc. However, I get error messages such as:
- Error CS1061 'CXCursor' does not contain a definition for 'Location'.
So, thinking it was the wrong version of ClangSharp, I updated to v. 8.0.0 beta. With this version I get error messages like:
- Error CS0246 The type or namespace name 'CXCursor' could not be found...
since they seemed to have renamed "CXCursor" to "Cursor" in the new version.
What version of ClangSharp are you using? It seems that neither version 3.8.0 or 8.0.0 of ClangSharp is compatible with the source code.

**Comment by xoofx** [comment]
> First, on the CppAst project no a target framework is set, and the option is grayed out and can't be set anyway.

It is normal, it is using the new csproj format which can [define multiple targets](https://github.com/xoofx/CppAst/blob/master/src/CppAst/CppAst.csproj#L3)

Are you sure that you have the latest VS2017/2019 installed and that you have also installed the latest .NET CORE Sdk just to make sure?

Also have you clone the repository recursively or checkout the submodule? (e.g `git submodule update --init --recursive`

It is likely a misconfiguration on your side.

**Comment by 0xc000005a** [comment]
The .NET Core 3.0 SDK was also installed, sorry I forgot to mention that. The VS 2019 was Version 16.3.5, I don't think there is a newer version. The VS 2017 I used probably wasn't the latest version though, but I'll stick with 2019 to reduce the # of possibilities. I downloaded the source from GitHub, rather than checking it out, so maybe that is causing the issues.

However, would that explain the differences in the ClangSharp versions {3.8, 8.0 } I tested, neither seemed compatible, but each had different issues in different locations...
